ob1k-cache/src/test/java/com/outbrain/ob1k/cache/MemcacheClientTest.java

```diff
@@ -35,7 +35,7 @@ public static void initialize() throws IOException {
       public String translateKey(final String key) {
         return key;
       }
-    }, 1, TimeUnit.MINUTES);
+    }, 1, TimeUnit.MINUTES, null);
```

Comment:
reject: I have marked the old API as @Deprocated as I do see much use for it in productive code - in this case of test code we would just send null.